polymer.externs.js

```diff
@@ -595,6 +595,15 @@ PolymerDomApi.prototype.setAttribute = function(attribute, value) {};
 /** @param {string} attribute */
 PolymerDomApi.prototype.removeAttribute = function(attribute) {};
 
+/**
+ * @param {Function} callback
```

Comment:
Here and below elsewhere: please add either "!" or "?" to explicitly specify nullability.